src/cryptoadvance/spectrum/spectrum.py

```diff
@@ -542,6 +542,28 @@ def testmempoolaccept(self, rawtxs, maxfeerate=0.1):
         return [{"allowed": True} for tx in rawtxs]
 
     @rpc
+    def getrawtransaction(self, txid, verbose=False):
+    """
```

Comment:
The build is complaining that the identation is not right here. And indeed it needs to be one tab on the right!

Reply:
Adding commits from phone was a bad idea. Let me get my hands on my laptop and I'll push a fix